pkg/networkservice/utils/metadata/helper/README.md

```diff
@@ -0,0 +1,69 @@
+# Metadata helper
+
+Metadata helper is a [genny](https://github.com/cheekybits/genny) template to generate a helper class for the metadata
+usage. Template parameters are:
+* prefix - prefix for the metadata type and constructor;
+* valueType - stored type.
+Generated helpers can both be exported (**P**refix) and unexported (**p**refix).
+
+gen.go:
+```go
+package connect
+
+//go:generate genny -in=../../utils/metadata/helper/meta_data.template.go -out=client_meta_data.gen.go -pkg=connect gen "prefix=client valueType=networkservice.NetworkServiceClient"
```

Comment:
Is there a way to reference the meta_data_template.go file that doesn't involve file system?  I ask, because this is all fine for using metadata inside the sdk repo... but would not work at all from the sdk-vpp repo...

Reply:
I have added `scripts/genny.sh`, so now we can get template by package + file name:
```go
//go:generate bash ../../../scripts/genny.sh -in=github.com/networkservicemesh/sdk/pkg/tools/metadatahelper/meta_data.template.go -out=int_meta_data.gen.go gen "prefix=Int valueType=int"
```
such script can be copied to other repositories.

Reply:
Clever solution :)